lib/internal/streams/destroy.js

```diff
@@ -108,7 +135,7 @@ function undestroy() {
   }
 }
 
-function errorOrDestroy(stream, err) {
+function errorOrDestroy(stream, err, sync) {
```

Comment:
`sync` means the opposite, perhaps rename to `async` or `nt`.

Reply:
Depends on how you interpret it. Are we in a "sync" context and therefore we need a "tick" or should we run "sync" (per your interpretation).

I called it "tick" before to make it less ambiguous. But "sync" is more consistent with the rest of the stream state code.

Reply:
OK. I wasn't aware of that interpretation, thanks.